tests/integrations/mcs/scheduling/api_test.go

```diff
@@ -218,3 +220,57 @@ func (suite *apiTestSuite) TestAPIForward() {
 		testutil.WithHeader(re, apiutil.ForwardToMicroServiceHeader, "true"))
 	re.NoError(err)
 }
+
+func TestAdminRegionCache(t *testing.T) {
```

Comment:
How about also checking if both caches can be cleaned in microservice mode?

Reply:
TestAdminRegionCacheForward tests it by "/pd/api/v1/admin/cache/region"

Reply:
What I mean is if we only call the API server's interface, both caches can be cleaned in microservice mode.